SAPex/Startup.cs

```diff
@@ -23,9 +32,11 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-               
             services.AddControllers();
             services.AddSwaggerGen();
+
+            // connection to server, look dbsettings
+            services.AddDbContext<AppDbContext>(options => options.UseSqlServer(confStringDb.GetConnectionString("DefaultConnection")));
         }
```

Comment:
in IConfiguration we have GetConnectionString read about this. and rewrite it with GetConnectionString  method